### DISCUSSION TITLE
Popover transition

### DISCUSSION THREAD

**Discussion by k4mr4n** [chakra-ui/chakra-ui#3157] [Q&A]
Hey All

I have a popover with heavy content and the transition makes it too slow. is it possible to disable the transition completely?

"@chakra-ui/react": "^1.1.5"

Thank you

**Comment by jonasgroendahl** [comment]
did you solve this?

**Reply by k4mr4n** [reply]
Yes, by dropping chakra and moving to tailwind instead.
Chakra is not optimized for big applications with lots of data.
https://chakra-ui.com/docs/comparison#the-runtime-trade-off-%EF%B8%8F
